server/config/logger.js

```diff
@@ -1,8 +1,56 @@
 const { createLogger, transports, format } = require("winston");
 
-class Logger {
-  constructor() {}
+// built in NPM levels.
+// {
+//   error: 0,
+//   warn: 1,
+//   info: 2,
+//   http: 3,
+//   verbose: 4,
+//   debug: 5,
+//   silly: 6
+// }
+
+const Loglevels = {
+  silly: 0,
+  verbose: 1,
+  info: 2,
+  debug: 3,
+  warn: 4,
+  error: 5,
+};
+const getDateFormat = () => {
+  let date = new Date().toDateString();
+  let time = new Date().toLocaleTimeString();
+  return `${date} || ${time}`;
+};
 
+class Logger {
+  constructor(lvl) {
```

Comment:
When we make a `new Logger` in `getInstance` we need to pass it the log level from ENV